### ISSUE TITLE
ISA: "BPF ADD" should be "BPF_ADD"

### ISSUE THREAD

**Issue by dthaler** [ebpffoundation/ebpf-docs#92]
```
``| BPF_ATOMIC | BPF_W  | BPF_STX |`` with 'imm' = BPF_ADD means::

  *(u32 *)(dst + offset) += src

``| BPF_ATOMIC | BPF_DW | BPF_STX |`` with 'imm' = BPF ADD means::

  *(u64 *)(dst + offset) += src
```

The "BPF ADD" should be `BPF_ADD` as in the first line.

**Comment by dthaler** [comment]
Patch posted at https://mailarchive.ietf.org/arch/msg/bpf/y3aNV8BEIY-RD0_4fn1R7h-mKho/

**Comment by dthaler** [comment]
Fixed in PR #99

**Comment by dthaler** [comment]
Fixed in draft -01
